src/models/User.ts

```diff
@@ -75,28 +75,50 @@ constructor(json: IUser) {
     */
     'patientId'?: string;
     /**
-    * Id of the patient if the user is a patient
-    */
+     * Id of the patient if the user is a patient
+     */
     'deviceId'?: string;
+  /**
+   * Delegations that are automatically generated client side when a new database object is created by this user
```

Comment:
`Ids of the dataOwners with who the user is sharing all new data he is creating in iCure : All Data Types that may be shared can be found in SharedDataType enum`